qemu/qemu.go

```diff
@@ -982,6 +1011,9 @@ const (
 	// VhostVSOCKPCI is the VSOCK vhost device type.
 	VhostVSOCKPCI = "vhost-vsock-pci"
 
+	// VhostVSOCKCCW is the VSOCK vhost device type for s390x.
+	VhostVSOCKCCW = "vhost-vsock-ccw"
+
```

Comment:
Generic vhost-vsock does not exist but to avoid unnecessary duplication, we can set VhostVSOCKPCI to "vhost-vsock-ccw" on Z. Not sure if the maintainers want this.